src/pudl_archiver/archivers/phmsagas.py

```diff
@@ -28,12 +30,25 @@ class PhmsaGasArchiver(AbstractDatasetArchiver):
 
     name = "phmsagas"
 
+    async def after_download(self) -> None:
+        """Clean up playwright once downloads are complete."""
+        await self.browser.close()
+        await self.playwright.stop()
```

Comment:
Noting for posterity this is the second time we're using this pattern; once we hit three we should refactor to pull out a central implementation